classrom.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './student'
+
+class Classroom
+  attr_accessor :label
+  attr_reader :student
```

Comment:
- [ ] Please change `student` to `student` in order to follow the Ruby naming conventions and prevent having errors.

Reply:
@MahmoudBakr23 Hi, can you clarify this one, please? As of now, it reads, Please change student to student. 

Thanks!

Reply:
@Amtresu Apologies, I meant `students` pluralized 😄